### PR TITLE
autodeploynode module doc

### PR DESCRIPTION
**Pull request by adean20csc**
This is the doc for the autodeploynode module, it replaces the previous config_autodeploy_node.

Removed the offline mode from the index.

### CODE REVIEW

File: docsite/source/autodeploynode_module.rst [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+
+Most commands require elevated privileges so you may need to :code:`su -`. Register
+with Red Hat Subscription Manager. Fill in the username and password with credentials
+that have a valid Red Hat subscription associated with it.
+
+.. code-block:: bash
+
+    su -
+    subscription-manager register --username=your_user --password=your_password
+
```
Comment (jdextercsc) [inline docsite/source/autodeploynode_module.rst]: If we are not offering offline mode, there is no reason to have OSP repos or subscriptions here.
```
+Find one of the repositories that include "Red Hat Openstack". Once a subscription
 +is found that provides Openstack note the "Pool ID"
 +
 +.. code-block:: bash
 +
 +    subscription-manager list --all --available
 +    subscription-manager attach --pool="Pool ID"
 +
 +Disable all repositories, then enable RPM repositories as needed.
 +
 +.. code-block:: bash
 +
 +    subscription-manager repos --disable=*
 +    subscription-manager repos --enable=rhel-7-server-rpms \
 +    --enable=rhel-7-server-optional-rpms \
 +    --enable=rhel-7-server-extras-rpms \
 +    --enable=rhel-7-server-openstack-6.0-rpms \
 +    --enable=rhel-server-rhscl-7-rpms \
 +    --enable=rhel-ha-for-rhel-7-server-rpms
```

File: docsite/source/autodeploynode_module.rst [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Next install the required support packages; ``epel-release, git and wget.``
+
+.. code-block:: bash
+
+    yum -y install https://dl.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-6.noarch.rpm
+    yum -y install git wget
+
+Install Ansible
```
Comment (jdextercsc) [inline docsite/source/autodeploynode_module.rst]: I thought we are going with epel again now that Ansible 2.0.1.x is in EPEL?

**Comment by mtnbikenc** [comment]
Offline code still exists, so we need the docs until we get that stuff pulled out.  Docs can be updated at the same time.

Review 👍

**Comment by jdextercsc** [comment]
Then Comments retracted 
+1

**Pull request merged**